include/gmock-win32.h

```diff
@@ -977,4 +984,5 @@ void patchModuleFunc_(void** mock_module_func_oldFn, TFunc func, TStub stub) {
     ::testing::Mock::VerifyAndClearExpectations(&mock_module_##func::instance());
 
 #define RESTORE_MODULE_FUNC(func) \
-    ::mockModule_restoreModuleFunc(*mock_module_##func::ppOldFn(), mock_module_##func::stub, mock_module_##func::ppOldFn())
+    gmock_win32::detail::restore_module_func( \
```

Comment:
The details in namespaces are good; It's easier to find everything through the IDE.

Reply:
yes, tried somehow to unify the place for such functions

Reply:
there is no doubt that this is the right way.